Resources/views/Block/block_search_result.html.twig

```diff
@@ -12,43 +12,52 @@ file that was distributed with this source code.
-                        <li><a href="{{ admin.generateObjectUrl('edit', result) }}">{{ admin.toString(result) }}</a></li>
+    <div class="col-md-4 col-sm-6">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">
+                    {% set icon = settings.icon|default('') %}
+                    {{ icon|raw }}
+
+                    {% if admin.hasRoute('list') %}
+                        <a href="{{ admin.generateUrl('list') }}">{{ admin.label|trans({}, admin.translationdomain) }}</a>
                     {% else %}
-                        <li><i>{{ admin.toString(result) }}</i></li>
+                        {{ admin.label|trans({}, admin.translationdomain) }}
                     {% endif %}
-                {% else %}
-                    {% if admin.hasRoute('create') %}
-                        <li><i>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }} ~ <a href="{{ admin.generateUrl('create') }}">{{ 'add_new_entry'|trans({}, 'SonataAdminBundle') }}</a></i></li>
+
+                    {% if pager and pager.getNbResults() > 0 %}
+                        <span class="badge pull-right">{{ pager.getNbResults() }}</span>
+                    {% endif %}
+                </h4>
+            </div>
+            <div class="panel-body">
+                <div class="list-group">
+                    {% if pager %}
+                        {% for result in pager.getResults() %}
+                            {% if admin.hasRoute('edit') %}
+                                <a href="{{ admin.generateObjectUrl('edit', result) }}" class="list-group-item">{{ admin.toString(result) }}</a>
+                            {% else %}
+                                <i class="list-group-item">{{ admin.toString(result) }}</i>
+                            {% endif %}
+                        {% else %}
+                            {% if admin.hasRoute('create') %}
+                                <span>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }}</span>
+                            {% else %}
+                                <span>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }}</span>
+                            {% endif %}
+                        {% endfor %}
+                        {% if pager.getNbResults() > pager.getResults()|length %}
+                            <span class="list-group-item"><i class="fa fa-ellipsis-h"></i></span>
+                        {% endif %}
                     {% else %}
-                        <li><i>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }}</i></li>
+                        {% if admin.hasRoute('create') %}
+                            <span>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }}</span>
+                        {% else %}
+                            <span>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }}</span>
+                        {% endif %}
```

Comment:
This looks redundant to me: if hasRoute() then 'no_results_found' else 'no_results_found'.
Did you forget something or can we reduce this to one ```<span>```?